### ISSUE TITLE
Incorrect comparison in apply_pka_values()

### ISSUE THREAD

**Issue by dvdesolve** [Electrostatics/pdb2pqr#131]
While preparing my protein with titration method = propka I've faced the following warning: `warning:pdb2pqr.protein pdb2pqr could not identify the following residues...` with a list of all troublesome residues. I've realized that the warning is false-positive because a key for comparison is pre-formatted with using exactly 1 space character, e. g. `key = "N+ %i %s" % (resnum, chain_id)`. However, `pkadic` contains keys with a multiple spaces so comparison fails and warning is issued. Possible fix at the very beginning of `apply_pka_values()` in `protein.py`:
```
        # rename dict keys so extra spaces will be omitted
        pkadic_new = dict()

        for key in pkadic:
            pkadic_new[" ".join(key.split())] = pkadic[key]

        pkadic = pkadic_new
```

**Comment by sobolevnrm** [comment]
Thank you for posting the fix with the problem!

**Comment by sobolevnrm** [comment]
Actually, it looks like #127 beat you to this. I fixed it a few days ago in #128; the fix is in the master branch now. Thanks again.
